monkestation/code/modules/slimecore/slime_traits/cleaner.dm

```diff
@@ -40,6 +40,7 @@
 	var/static/list/huntable_trash = typecacheof(list(
 		/obj/item/trash,
 		/obj/item/food/deadmouse,
+		/obj/item/shard
 	))
 
```

Comment:
Some things I'd suggest adding (to the relevant typecaches):
- `/obj/item/cigbutt`
- `/obj/item/food/candy_trash`
- spent ammo casings - `/obj/item/ammo_casing`, but only if `loaded_projectile` is null/deleted)
- certain storage items (i.e `/obj/item/storage/box/foodpack`, `/obj/item/storage/fancy/cigarettes`) if they're empty or only contain empty/trash items
- `/obj/item/food/breadslice/moldy`
- `/obj/item/light/tube/broken` and `/obj/item/light/bulb/broken`
- `/obj/effect/decal/cleanable/piss_stain`
  - DON'T THINK ABOUT THIS TOO HARD
- `/obj/item/popsicle_stick`
- `/obj/item/broken_bottle`

if you want tho, i can make myself as a separate PR later, rather than have it bundled up into this one.

Reply:
added everything that doesnt have a special condition associated

Reply:
not pushed yet, testing other changes